kolibri/plugins/coach/assets/src/views/groups-page/group-section.vue

```diff
@@ -68,7 +68,7 @@
               :showLabel="false"
               :checked="isSelected(user.id)"
               @change="toggleSelection(user.id)"
-              @click.stop
+              @click.native.stop
```

Comment:
Cool. What does `native` do here?

Reply:
Catches the native click event of the component, since the k-checkbox does not actually emit a click event. https://vuejs.org/v2/guide/components.html#Binding-Native-Events-to-Components

Reply:
Do you think it would be possible to make `@change` on the `k-checkbox` a no-op because the outer div also has `toggleSelection(user.id)` as its on-click callback too? Or does the outer div not register a click when you hit the checkbox exactly?

Reply:
The input element of the k-checkbox has ` @click.stop="toggleCheck"` to stop the event propagation and prevent the checkbox container from also triggering `toggleCheck`. So clicking directly on on the checkbox, stops the click event from bubbling up and ever making reaching `@click="toggleSelection(user.id)"`

Reply:
So not possible.